### PR TITLE
Adds *chirp for birdpeople

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -29,6 +29,18 @@
 				message = "<B>[src]</B> flaps \his wings ANGRILY!"
 				m_type = 2
 
+		if ("chirp")
+			if (src.dna.species.id == "avian")
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/human/emote.dm]: "src." is useless,remove it

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -29,6 +29,18 @@
...
+				var/sound = pick("birdchirp.ogg", "birdchirp2.ogg", "birdchirp3.ogg", "birdchirp4.ogg", "birdchirp5.ogg", "birdchirp6.ogg", "birdchirp7.ogg", "birdchirp8.ogg", "birdchirp9.ogg", "birdchirp10.ogg", "birdchirp11.ogg", "birdchirp12.ogg")
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/human/emote.dm]: Files need single quotes ie '     '

**Comment by ghost** [comment]
oh wow I'm stupid, fixed. Might as well remove the other srcs, too

**Comment by ghost** [comment]
fixed.

**Comment by ghost** [comment]
fucking travis reee

**Comment by ghost** [comment]
well fuck

**Comment by Spacedong** [comment]
kek

**Comment by ghost** [comment]
will fix the travis later after my computer stops being a cuck

are headcoders cool with this?

**Comment by GenericGoose** [comment]
do they chirp when you hang them
Birds are the greatest thing to lynch 
Because when they're up there they cry like a little bitch 
I'm the one to dig the ditch

**Comment by ghost** [comment]
@GeneralGoose yes

**Comment by GenericGoose** [comment]
i rate 👌👌👌👌👌👌👌👌👌👌/10

**Comment by ghost** [comment]
> 12 oggs over meme birds.

Most of them have background noise or multiple birds chirping.
Even then, I'm not a huge fan of adding REAL sounds to spess.

**Comment by SplergeBucket** [comment]
I don't want any sounds which have shit quality with background noise.
Travis failed anyway.

**Comment by Carbonhell** [comment]
Two headmins against and the PR isn't even working. Closing.

**Pull request closed without merging**